docs/tutorials/optimizers_conditionalgradient.ipynb

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
<p>Could you refactor this notebook to match the new <code>_template.ipynb</code>:</p><p><a href="https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb" target="_blank">https://github.com/tensorflow/addons/blob/master/docs/tutorials/_template.ipynb</a></p><p> </p><p>Should have a download notebook and tensorflow.org link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>I have added the tag for download notebook and TensorFlow.org. But I have not put href link into it. I am not exact sure for the link address to put in for download and Tensorflow.org part. Sorry about this.</p><p> </p>

Reply:
See updated comments